imageprocessor/src/main/java/pl/edu/uj/kimage/processingFlow/FlowStep.java

```diff
@@ -0,0 +1,42 @@
+package pl.edu.uj.kimage.processingFlow;
+
+import pl.edu.uj.kimage.Plugin;
+import pl.edu.uj.kimage.eventbus.Event;
+
+public class FlowStep {
```

Comment:
What is the difference between Step and FlowStep? I'm not sure logic correctness, cause there is no usage of this code...